.github/workflows/trigger_release.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger Release
```

Comment:
Will this always create a `prerelease`?

Reply:
yes I'll change the name, an other chronjob will in the future upgrade the pre-release to release